src/transactions/typeTag/index.ts

```diff
@@ -340,6 +340,12 @@ export class TypeTagStruct extends TypeTag {
   }
 }
 
+export class TypeTagString extends TypeTagStruct {
+  constructor() {
+    super(new StructTag(AccountAddress.ONE, new Identifier("string"), new Identifier("String"), []));
+  }
+}
```

Comment:
There's a type for this?  `stringTypeTag()`

It's not an actual TypeTag to be a string, so... it'll get real confusing really quick to have types that don't match the underlying data.  Not to mention we already have fragmentation between SDKs

Reply:
Each `TypeTag` type represents an actual underlying type in the rust enum `TypeTag` used for the MoveVM.  Each one has a different encoding associated.

String is not a native type in Move, it's a `Struct` type tag, with specific properties.